packs/bestiary-family-ability-glossary/cryptid-primeval-grasp-for-life.json

```diff
@@ -10,6 +10,7 @@
             "value": null
         },
         "category": "offensive",
+        "deathNote": true,
         "deathnote": true,
```

Comment:
```suggestion
```
This is the one to delete ("n" in Note should be capitalized, which is why it doubled up)

Reply:
Ah, good catch. We can fix that on a follow up.